docs/source/install/index.rst

```diff
@@ -60,13 +60,38 @@ For Fedora/RHEL/CentOS users, run:
 
    sudo yum install -y gcc-c++ python3-devel make cmake
 
-Create conda environment for development, run:
+To create a Conda environment for CPU development, please execute the following command:
+
+.. code:: bash
+
+   bash script/create_dev_conda_env.sh -c
+
+To create a Conda environment for GPU development, please execute the following command:
+
+.. code:: bash
+
+   bash script/create_dev_conda_env.sh -g 11.7
+
+
+To see the usage of create script, run:
```

Comment:
To further configure the conda environment, please execute the following command for more details:

Reply:
OK, I changed it.